packages/auth-helpers/package.json

```diff
@@ -17,6 +17,9 @@
     "access": "public",
     "directory": "../../dist/packages/auth-helpers"
   },
+  "peerDependencies": {
+    "tslib": "^2.3.0"
+  },
```

Comment:
we might want to remove the `^` in the version if versions ABOVE 2.3.0 are not intended to be used.

Reply:
This is consistent with the root package.json's dep of tslib being ^2.3.0, and, just tested it in a client app which installed 2.5.0 and it works!